packages/node_modules/@webex/widget-recents/src/selector.js

```diff
@@ -27,6 +29,17 @@ const getMercuryStatus = (state) => state.mercury.get('status');
 const getFeatures = (state) => state.features;
 const getWidgetProps = (state, props) => props;
 
+const getSDKAdapter = createSelector(
+  [getSpark],
```

Comment:
Does this create a new sdk adapter every time the spark module changes?

Reply:
This creates a new SDK adapter every time there is a new spark module instantiated.